solarkraft/src/verify.ts

```diff
@@ -103,6 +108,50 @@ function apalacheCheck(monitor: string): Result<ApalacheResult> {
     }
 }
 
+// For now, our solution is TESTNET-ONLY!, so
```

Comment:
Maybe file an issue with the enhancement label?
That's how I'm keeping track of stuff that's fine for now, but should eventually be resolved.